### ISSUE TITLE
Any possibility to get export to Postman collection

### ISSUE THREAD

**Issue by mayankkushal** [rangav/thunder-client-support#162]
So its easier to share with people who aren't using this wonderful extension yet

**Comment by rangav** [comment]
Hi @mayankkushal good question, it’s not in our roadmap. 

That is something postman should build. Or write you own json converter.

**Comment by mssalnikov** [comment]
This is a bummer, my team uses postman, and I made all new API requests in ThunderClient, and now can't share it with them( This is a very important feature because it blocks using thunder.

Is there at least a way to manually convert collections? Posman gives the error "Collections v1 are not supported anymore", but using [this tool](https://learning.postman.com/docs/getting-started/importing-and-exporting-data/#converting-postman-collections-from-v1-to-v2) on thunder export  creates an empty postman collection 

At least this way 
`postman-collection-transformer convert -i thunder-collection.json -o postman-collection.json -j 1.0.0 -p 2.0.0 -P`

**Comment by mssalnikov** [comment]
@rangav I don't agree this is something postman should build. Thunder client is supposed to be a postman replacement, which means, it should be interoperable, otherwise, you just can't expect people who are used to postman to use this extension, since they're forced to either make all their colleagues use it, or don't use it at all. And not using it at all is far easier than changing minds of _all colleagues_ at the same time, and not everyone is using vscode

**Comment by rangav** [comment]
Hi @mssalnikov got it will think about it.

**Comment by fernandohonig** [comment]
Hi @rangav thanks for this plugin first of all. I think it's a great plugin to local development, but when you move to a CI environment, using newman, would be great to have the option to export from your local Thunder client to a newman compatible forman (aka Postman v2.x)
This feature would definitely get more people into Thunder Client as they will not need to use postman anymore neither for local or CI testing approach.

**Comment by teocci** [comment]
I was working for an entire week using Thunder client in the testing server when I moved to the production server that is isolated in another DMZ I was not able to pass the collection I made an entire week. This a vital feature if you really want to claim that is the postman replacement. And clearly it is not you can not even share collections between Thunder Clients.

**Comment by rangav** [comment]
Hi @teocci , this feature is available from the first version.  

1. From the collections option menu (…) click export.  

2. Click Menu icon in Collections tab, you will see Import option

**Comment by rangav** [comment]
Hi All, This feature will be part of next update - v1.12.0

Note: not all data is compatible with postman format.

- Tests will not be converted
- Collection Settings
- May have some issue with path variables

**Comment by rangav** [comment]
This feature is now implemented in v1.12.0

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v1.12.0

please let me know if any feedback

**Comment by rangav** [comment]
read point 3 here for details 
https://github.com/rangav/thunder-client-support/releases/tag/v1.12.0
